pages/_app.js

```diff
@@ -119,6 +120,18 @@ class OpenCollectiveFrontendApp extends App {
         }
         window._paq.push(['trackPageView']);
       }
+      this.setState(state => {
+        const history =
+          url === state.history.slice(-1)[0]
+            ? // Do not repeat the same URL
+              state.history
+            : url === state.history.slice(-2)[0]
+              ? // If returning, pop the last URL
+                state.history.slice(0, -1)
+              : // Else, keep the last 10 URLs visited
+                [...state.history, url].slice(-10);
+        return { history };
```

Comment:
:ice_cream: Personal preference: I tend to find if/else if blocks cleaners than nested ternaries for this kind of long condition.